volontulo/templates/users/user_profile.html

```diff
@@ -4,59 +4,83 @@
 {% block title %}Strona użytkownika {{ user.email }}{% endblock %}
 
 {% block content-heading %}
-    <div class="media panel panel-default">
-        <div class="panel-heading media-heading lead">Użytkownik: {{ user.email }}</div>
-        <div class="panel-body">
-            <div class="media-left">
-                <p><img src="{{ MEDIA_URL }}{{ userprofile.get_avatar.0.image }}" alt="{{ user.username }}" class="img-circle" width="100" height="100" /></p>
-            </div>
-            <div class="media-body media-middle">
-                <ul class="list-unstyled">
-                    <li><b>Real User ID:</b> <span class="badge">{{ user.id }}</span></li>
-                    <li><b>UserProfile ID:</b> <span class="badge">{{ user.userprofile.id }}</span></li>
-                </ul>
-            </div>
+    <div class="media">
+        <div class="media-left">
+            <img class="img-responsive" src="{{ MEDIA_URL }}{{ userprofile.get_avatar.0.image }}" alt="{{ user.username }}" />
+        </div>
+        <div class="media-body media-top">
+            <h2 class="lead">Użytkownik: {{ user.username }}</h2>
+            <ul class="list-unstyled">
+                <li><b>Real User ID:</b> <span class="badge">{{ user.id }}</span></li>
+                <li><b>UserProfile ID:</b> <span class="badge">{{ user.userprofile.id }}</span></li>
+            </ul>
+        </div>
+        <div class="media-body media-top">
```

Comment:
What do you think about moving this HTML wraps into common/badges.html? That will allow us to remove badges without leaving empty div elements.